modules/www/templates/code-submitter.env.erb

```diff
@@ -1 +1 @@
-AUTH_BACKEND={"backend":"code_submitter.auth.NemesisBackend","kwargs":{"url":"https://<%= @fqdn %>/userman/","verify":<%= @verify_tls %>}}
+AUTH_BACKEND={"backend":"code_submitter.auth.FileBackend","kwargs":{"path":"/srv/secrets/code-submitter-credentials.yaml"}}
```

Comment:
I don't think we should have this read directly from `/srv/secrets`. For one thing, the server process definitely should not have access to that directory.